### ISSUE TITLE
Crashes homebridge if can't connect

### ISSUE THREAD

**Issue by christophersjchow** [rgavril/homebridge-xiaomi-air-purifier3#7]
This seems to crash Homebridge if it can't connect.

```
Mar 01 20:07:28 homebridge homebridge[3199]: MIoT Device (did: xxx) failed to connect:Error: Could not connect to device, handshake timeout
Mar 01 20:07:28 homebridge homebridge[3199]: MIoT Device (did: xxx) failed to connect:Error: Could not connect to device, handshake timeout
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getAirQuality
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getAirQuality Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getLockPhysicalControls
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getLockPhysicalControls Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getCurrentAirPurifierState
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getCurrentAirPurifierState Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getTargetAirPurifierState
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getTargetAirPurifierState Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getRotationSpeed
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getRotationSpeed Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getActive
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getActive Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getTemperature
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getTemperature Failed: MIOTDevice property has unknown value.
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getHumidity
Mar 01 20:07:39 homebridge homebridge[3199]: [3/1/2020, 8:07:39 PM] [Master Air Purifier] getHumidity Failed: MIOTDevice property has unknown value.
Mar 01 20:07:54 homebridge homebridge[3199]: [3/1/2020, 8:07:54 PM] [Master Air Purifier] Polling properties
Mar 01 20:07:54 homebridge homebridge[3199]: [3/1/2020, 8:07:54 PM] undefined
Mar 01 20:07:54 homebridge homebridge[3199]: [3/1/2020, 8:07:54 PM] Got SIGTERM, shutting down Homebridge...
Mar 01 20:07:55 homebridge homebridge[3199]: [3/1/2020, 8:07:55 PM] undefined
Mar 01 20:07:59 homebridge systemd[1]:
```

I'm running the latest homebridge and the only other plugin I'm using is the Wemo Platform.

**Comment by mksmo** [comment]
I've got the same problem.

**Comment by rgavril** [comment]
Thanks to reporting. 

The problem was fixed, please update to 1.2.1
